zkevm-circuits/src/evm_circuit/util/common_gadget/curie.rs

```diff
@@ -41,18 +41,18 @@ impl<F: Field> CurieGadget<F> {
         );
 
         // TODO: refactor
-        // is_scoll_chain means (chain_id - 534352) * (chain_id - 222222) == 0
-        let is_scroll_chain = IsZeroGadget::construct(
+        // is_morph_chain means (chain_id - 53077) * (chain_id - 2710) == 0
+        let is_morph_chain = IsZeroGadget::construct(
```

Comment:
Maybe you can try https://github.com/scroll-tech/zkevm-circuits/blob/84b9bb5618f7041f0ff364a89d305ded2c0ab865/zkevm-circuits/src/evm_circuit/util/math_gadget/batched_is_zero.rs#L19, for batch constraint chain id.